nltk/translate/bleu_score.py

```diff
@@ -150,8 +150,8 @@ def corpus_bleu(
     :type list_of_references: list(list(list(str)))
     :param hypotheses: a list of hypothesis sentences
     :type hypotheses: list(list(str))
-    :param weights: weights for unigrams, bigrams, trigrams and so on
-    :type weights: list(float)
+    :param weights: weights for unigrams, bigrams, trigrams and so on, (one or list of weights)
```

Comment:
```suggestion
    :param weights: weights for unigrams, bigrams, trigrams and so on (one or a list of weights)
```
Perhaps this flows a bit better.

Reply:
Fixed in c730dc3. I'm open to suggestions or changes.